### DISCUSSION TITLE
VS Code Astro formatting not working on a specific project

### DISCUSSION THREAD

**Discussion by nayaabkhan** [withastro/language-tools#880] [Support]
Hey fellows,

I have an Astro project for which the Astro extension isn't able to format the `.astro` files. The rest of the features such as intellisense work fine.

On the other Astro projects, the formatter works fine. Not able to pin down the cause.

Is there a way I can see the logs of the formatter used by the extension to debug this further?

**Comment by Princesseuh** [comment]
Hello, you can see the logs in the "Output" panel. You should be able to filter for "Astro" on the right of it. (same place where the terminal is)

**Comment by StevenStavrakis** [comment]
I am having the same issue. Nothing appears in my output panel when I run formatting.

**Comment by FaydSpeare** [comment]
I find that the formatting only works if I use the folder containing Astro and prettier config as the root of the project in VSCode.

**Reply by StevenStavrakis** [reply]
That doesn't seem to make a difference for me

**Comment by StevenStavrakis** [comment]
I deleted:
```
"[astro]": {
  "editor.defaultFormatter": "astro-build.astro-vscode"
}
```

From my workspace .vscode folder and now the formatting appears to be working.
